spring-batch/src/test/java/com/baeldung/taskletsvschunks/chunks/ChunksIntegrationTest.java

```diff
@@ -1,25 +1,31 @@
 package com.baeldung.taskletsvschunks.chunks;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 import com.baeldung.taskletsvschunks.config.ChunksConfig;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+
+import org.junit.jupiter.api.Test;
 import org.springframework.batch.core.ExitStatus;
 import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.JobParameters;
 import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.batch.test.context.SpringBatchTest;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(classes  = ChunksConfig.class)
+@SpringBatchTest
+@EnableAutoConfiguration
+@ContextConfiguration(classes = ChunksConfig.class)
 public class ChunksIntegrationTest {
 
-    @Autowired private JobLauncherTestUtils jobLauncherTestUtils;
+    @Autowired
+    private JobLauncherTestUtils jobLauncherTestUtils;
 
     @Test
     public void givenChunksJob_WhenJobEnds_ThenStatusCompleted() throws Exception {
-        JobExecution jobExecution = jobLauncherTestUtils.launchJob();
-        Assert.assertEquals(ExitStatus.COMPLETED, jobExecution.getExitStatus());
+        JobParameters jobParameters = this.jobLauncherTestUtils.getUniqueJobParameters();
```

Comment:
Why is line 27 needed here?

Reply:
It was a test, I reverted to the previous version.